pyproject.toml

```diff
@@ -22,6 +22,7 @@ aiohttp = "~3.7.3"
 urllib3 = ">=1.25.3"
 feedparser = "^6.0.2"
 regex = "^2020.11.13"
+googletrans = "==3.1.0a0"
```

Comment:
I'm a little worried as to where this version comes from on PyPI, since the master branch on the repo is 3.0.0 – any ideas?